waiter/integration/waiter/request_timeout_test.clj

```diff
@@ -196,21 +196,21 @@
                                       :name (str "test" token)
                                       :grace-period-secs (t/in-seconds grace-period)
                                       :cmd-type "shell"})]
-          (is (= 200 status) (str "Did not get a 200 response. " body)))
+          (assert-response-status response 200))
         (log/info "Making request for" token)
         (let [{:keys [status body service-id] :as response} (make-request-with-debug-info
                                                               {:x-waiter-token token}
-                                                              #(make-request waiter-url "/secrun" :headers %))
-              instance-acquired-delay-ms (-> response
-                                             :headers
-                                             (get "x-waiter-get-available-instance-ns")
-                                             Long/parseLong
-                                             (quot 1000000))] ; truncated nanos->millis
-          (is (= 200 status) (str "Did not get a 200 response. " body))
-          (is (<= startup-delay-ms instance-acquired-delay-ms)
-              (format "Healthy instance was found in just %dms (too short)" instance-acquired-delay-ms))
+                                                              #(make-request waiter-url "/secrun" :headers %))]
+          (assert-response-status response 200)
           (when (and (= 200 status) (can-query-for-grace-period? waiter-url))
```

Comment:
`can-query-for-grace-period?` current implementation means we will not check the following assertions for kubernetes scheduler, is that intended?